src/txmempool.h

```diff
@@ -605,11 +605,9 @@ class CTxMemPool
      *                                          to mempool. The transactions need not be direct
      *                                          ancestors/descendants of each other.
      * @param[in]       total_vsize             Sum of virtual sizes for all transactions in package.
-     * @param[out]      errString               Populated with error reason if a limit is hit.
```

Comment:
```cpp
* @returns {} or the error reason if a limit is hit.
```

Reply:
this would be good

Reply:
Thanks Would create a tiny follow-up to fix this, so as not to invalidate ACK's.

Reply:
This is fixed in https://github.com/bitcoin/bitcoin/pull/29115